src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java

```diff
@@ -57,6 +64,8 @@ public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalsPoll
   }
 
   public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalQueueRecordFlow(Configuration configuration) {
+    log.info("registerRetrievalQueueRecordFlow:: Registering retrieval queue record flow for configuration: {}", configuration.getId());
```

Comment:
Please change this to debug and keep single logger statement as method itself is having single line.

Reply:
Changed to debug.